dbms/src/test/java/org/polypheny/db/sql/view/ComplexViewTest.java

```diff
@@ -2024,7 +1806,7 @@ public void testQ19() throws SQLException {
 
     // SELECT NOT POSSIBLE
     // java.lang.AssertionError: type mismatch: ref: VARCHAR(25) NOT NULL input: INTEGER NOT NULL
-    @Ignore
+    @Disabled
```

Comment:
Can you enable this test?

Reply:
Yes, after https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1519992631, enabled